site/en/reference/cli/concepts/srp.md

```diff
@@ -0,0 +1,296 @@
+# Set Up Service Registration Protocol (SRP) Server-Client Connectivity With OT CLI
+
+OpenThread offers SRP server and client commands for use with a Thread network
+to perform such functions as registering and leasing services, and managing
+server and client hosts.
```

Comment:
We should simplify/change this. There is no notion of "leasing services" in SRP. 

Suggest the following:
OpenThread offers both SRP server and client functionality, enabling devices to register DNS-based services using the standard DNS Update sent as unicast packets. This functionality enables DNS-Based Service Discovery.